src/main_thread/api/debug/buffer_graph.ts

```diff
@@ -28,7 +28,7 @@ export interface ISegmentSinkGrapUpdateData {
 
 export default class SegmentSinkGraph {
   /** Link buffered Representation to their corresponding color. */
-  private readonly _colorMap: WeakMap<IRepresentation, string>;
+  private readonly _colorMap: Map<string, string>;
```

Comment:
Maybe we could update the comment to indicate that it's the Representation's `uniqueId`